applications/dot_marketplace-Phase3.md

```diff
@@ -0,0 +1,222 @@
+
+| Sr no. | Deliverable | Description |
+| --- | --- | --- |
+| 0a | License | Apache 2.0 |
+| 0b | Documentation | We will provide both inline documentation of the code and a tutorial that explains how a user can use DOT Marketplace and understand the flow of tasking pallet. |
+| 0c | Testing Guide | Functions will be covered by unit tests, the documentation will describe how to run these tests. We will also provide scripts to help deploy, run and test the build. |
+| 0d | Docker Image | Docker image of the build |
+| 1 | Decentralized Court Module | An on-chain decentralized court for dispute resolution within the ecosystem. |
+| 1a | Disapprove Milestone  | In the case of a customer not being satisfied with the work submitted by the service provider (worker). A set of jurors is shortlisted (court summon) to resolve the dispute and pass a verdict. |
+| 1b | Disapprove Rating | The customer or the service provider, once they have received their rating for a particular milestone and are not satisfied with it. |
+| 1c | General Dispute | A general dispute function for cases that do not fall under the categories mentioned in 1a and 1b. |
+| 2 | Voting module | Each juror can review the dispute and cast their vote, which also includes their rating for both the customer and the worker. After two days, all the juror votes are counted, and a winner is identified. |
+| 3 | Frontend App  | Supporting frontend UI to test the aforementioned functionality. |
+
+### **Milestone 3**
+
+* **Estimated duration:** 3.0 weeks
+* **FTE:** 1
+* **PTE:** 2
+* **Costs:** 7975 USD
+
+
+The main deliverables in this milestone are to use decentralized IPFS based storages to store all the files realated to tasks & advanced search. A file server integrated to nft storage will provided, using rocket framework & the search feature will be an update to the makerplace pallet. The skeletal UI will also be updated to showcase all the new features in Phase3.
+
+| Sr no. | Deliverable | Description |
+| --- | --- | --- |
+| 0a | License | Apache 2.0 |
+| 0b | Documentation | We will provide both inline documentation of the code and a tutorial that explains how a user can use DOT Marketplace and understand the flow of tasking pallet. |
+| 0c | Testing Guide | Functions will be covered by unit tests, the documentation will describe how to run these tests. We will also provide scripts to help deploy, run and test the build. |
+| 0d | Docker Image | Docker image of the build |
+| 1 | Decentralized Stroage  | All tasks related docs will be stored on a decentralized IPFS platform. |
+| 2 | Advanced Search  | Search based on task progress, tags, tasks or milestone id's. |
+| 3 | Frontend App  | Supporting frontend UI to test the aforementioned functionality. |
```

Comment:
Same here - Can you provide wireframes?

Reply:
We will be providing a skeletal UI to showcase the features of dot_marketplace. [Phase1 Guide](https://github.com/WowLabz/dot_marketplace_docker/tree/Phase1_Milestone3]) has the UI screenshots for the same. This will be updated for the Phase3 functionalies.